src/executables/server.c

```diff
@@ -403,27 +406,20 @@ main (int argc, char **argv)
   {				/* to make indent happy */
     if (argc < 2)
       {
-        PRINT_AND_LOG_ERR_MSG ("Usage: server databasename\n");
-        return 1;
+	PRINT_AND_LOG_ERR_MSG ("Usage: server databasename\n");
+	return 1;
```

Comment:
Is there a reason to return 1/0?
```suggestion
	return ER_FAILED;
```
and `NO_ERROR` at the end

Reply:
The main function returns an exit code, not the internal database error code. Other main functions return `EXIT_SUCCESS` or `EXIT_FAILURE`. They may be used here too to replace 1/0.